fix(curriculum): separate unreachable code and scope for pyramid project (#54920)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-introductory-javascript-by-building-a-pyramid-generator/6610c77d50636722e5b6be17.md

# --description--

Below the `return` statement, log the string `"This works!"` to the console.

After doing that, you will see that the string `"This works!"` does not display in the console, and the `console.log("This works!")` line is greyed out.

> Copy and paste the line of code above the `return` statement. Now, the string `"This works!"` should appear in the console.

Reviewer: ```suggestion
Copy the console log and paste it above the `return` statement. Now, the string `"This works!"` should appear in the console.
```

# --description--

> Add `console.log("This works!");` below the `return` statement.

Reviewer 2: ```suggestion
Below the `return` statement, log the string `"This works!"` to the console.
```

> There is a bit of a problem, here. Your `console.log` is currently after your `return` statement.
> The string `"This works!"` does not display in the console, and the newly placed code appears washed out.
> Copy and paste the `console.log` above the `return` statement.
> Now the string `"This works!"` appears in the console.
> 
> An important thing to know about the `return` keyword is that it does not just define a value to be returned from your function, it also stops the execution of your function code. This means that any code after a `return` statement will not run.

Reviewer 2: ```suggestion
After doing that, you will see that the string `"This works!"` does not display in the console, and the `console.log("This works!")` line is greyed out.

Copy and paste the line of code above the `return` statement. Now, the string `"This works!"` should appear in the console.

An important thing to know about the `return` keyword is that it does not just define a value to be returned from your function, it also stops the execution of your code inside a function or a block statement. This means any code after a `return` statement will not run.
```
